core/templates/pages/admin-page/config-tab/admin-config-tab.directive.html

```diff
@@ -26,3 +26,15 @@ <h3>Configuration properties</h3>
     No configuration properties are available.
   </div>
 </md-card>
+
+<style>
+  .oppia-long-text-extra {
+    max-width: 1050px;
```

Comment:
The same class has different max-widths everywhere. Shouldn't you scope these classes down to this directive only? As otherwise, it might affect other uses of this class?